x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -264,19 +269,21 @@ export const getManagementFilteredLinks = async (
       linksToExclude.push(SecurityPageName.responseActionsHistory);
     }
 
-    if (!canIsolateHost && canUnIsolateHost) {
-      let shouldSeeHIEToBeAbleToDeleteEntries: boolean;
+    if (canIsolateHost && !hasPlatinumLicense) {
+      let shouldBeAbleToViewAndDeleteHIEArtifacts: boolean;
       try {
         const hostExceptionCount = await getHostIsolationExceptionTotal(core.http);
-        shouldSeeHIEToBeAbleToDeleteEntries = hostExceptionCount !== 0;
+        // has an HIE entry and is a super user then set to TRUE
+        shouldBeAbleToViewAndDeleteHIEArtifacts =
+          hostExceptionCount !== 0 && canAccessEndpointManagement;
       } catch {
-        shouldSeeHIEToBeAbleToDeleteEntries = false;
+        shouldBeAbleToViewAndDeleteHIEArtifacts = false;
       }
 
```

Comment:
instead of doing this logic here, can I suggest that you refactor the hook I pointed to earlier adn extract out an `async` function that does the checks adn then use it here (and should also be used in the hook). Would suggest NOT having mulitiple areas of code all doing calculation for Authz and instead having just one piece of code that handles that logic.